tests/unit/docs/test_service.py

```diff
@@ -111,3 +112,20 @@ def test_document_service_no_waiter(self):
         service_documenter = ServiceDocumenter('myservice', self.session)
         contents = service_documenter.document_service().decode('utf-8')
         self.assertNotIn('Waiters', contents)
+
+    def test_creates_correct_path_to_examples_based_on_service_name(self):
+        path = os.sep.join([os.path.dirname(__file__), '..', '..', '..',
```

Comment:
I would import boto3 and use ``boto3.__file__`` to determine this location instead of relying on the location of the test. That way if the test ever moves location, we do not have to update the test.